source/routing/raptor.h

```diff
@@ -140,7 +140,8 @@ struct RAPTOR
                 const std::vector<std::string>& forbidden = std::vector<std::string>(),
                 bool clockwise = true,
                 const boost::optional<navitia::time_duration>& direct_path_dur = boost::none,
-                const size_t max_extra_second_pass = 0);
+                const size_t max_extra_second_pass = 0,
+                const navitia::time_duration& transfer_penalty = 2_min); // "les 2 minutes du peuple"
```

Comment:
We will probably have a struct containing all params at one point.
(And deal with exposition of this param in jormun also).
So we'll let it that way for this PR.